ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -250,6 +250,11 @@
     reason="Test runs ONLY on OSD or ROSA cluster",
 )
 
+hci_required = pytest.mark.skipif(
```

Comment:
Does https://github.com/red-hat-storage/ocs-ci/blob/1af3c3b2cf7fd7bbb8c78cc1b8bbf4d2480870b3/ocs_ci/framework/pytest_customization/marks.py#L318 not work with hci_provider_and_client_required

Reply:
Yes, you're right, it would work as well.

Reply:
The setup is invalid when no provider or no client is present, right? Maybe it would be better to have only one marker for hci platform to prevent redundancy.

Reply:
Fixed